packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -938,6 +938,17 @@ describe(componentName, () => {
       unobserve: jest.fn(),
       disconnect: jest.fn(),
     }));
+    Object.defineProperty(window, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation((query) => ({
+        matches: false,
+        media: query,
+        onchange: null,
+        addEventListener: jest.fn(),
+        removeEventListener: jest.fn(),
+        dispatchEvent: jest.fn(),
+      })),
+    });
```

Comment:
What is this test for? Why was this added in this Framer motion PR? Just curious 🧐

Reply:
Good question, this just mocks `window.matchMedia` which is used within our local `usePrefersReducedMotion` hook. I'm not sure how the `useReducedMotion` hook is working from `framer-motion` but because ours uses `window.matchMedia` we need to mock it in jest (now that `FilterPanel` from Datagrid is using it).

Otherwise, we see test failures like this...
![image](https://github.com/user-attachments/assets/083c88d2-f712-489a-af64-4d5ebb7ca7a5)